vectorstores/alloydb/vectorstore_test.go

```diff
@@ -13,76 +13,108 @@ import (
 	if projectID == "" {
-		t.Skip("ALLOYDB_PROJECT_ID environment variable not set")
+		t.Skip("env variable PROJECT_ID is empty")
 	}
+	// Requires environment variable ALLOYDB_REGION to be set.
 	region := os.Getenv("ALLOYDB_REGION")
 	if region == "" {
-		t.Skip("ALLOYDB_REGION environment variable not set")
+		t.Skip("env variable ALLOYDB_REGION is empty")
 	}
+	// Requires environment variable ALLOYDB_INSTANCE to be set.
 	instance := os.Getenv("ALLOYDB_INSTANCE")
 	if instance == "" {
-		t.Skip("ALLOYDB_INSTANCE environment variable not set")
+		t.Skip("env variable ALLOYDB_INSTANCE is empty")
 	}
+	// Requires environment variable ALLOYDB_CLUSTER to be set.
 	cluster := os.Getenv("ALLOYDB_CLUSTER")
 	if cluster == "" {
-		t.Skip("ALLOYDB_CLUSTER environment variable not set")
+		t.Skip("env variable ALLOYDB_CLUSTER is empty")
+	}
+	// Requires environment variable ALLOYDB_TABLE to be set.
+	table := os.Getenv("ALLOYDB_TABLE")
+	if table == "" {
+		t.Skip("env variable ALLOYDB_TABLE is empty")
 	}
 
-	return username, password, database, projectID, region, instance, cluster, table
+	// Requires environment variable GOOGLE_CLOUD_LOCATION to be set.
+	location := os.Getenv("GOOGLE_CLOUD_LOCATION")
+	if location == "" {
+		t.Skip("env variable GOOGLE_CLOUD_LOCATION is empty")
+	}
+
+	envVariables := EnvVariables{
+		Username:  username,
+		Password:  password,
+		Database:  database,
+		ProjectID: projectID,
+		Region:    region,
+		Instance:  instance,
+		Cluster:   cluster,
+		Table:     table,
+	}
+
+	return envVariables
 }
 
-func setEngine(t *testing.T) (alloydbutil.PostgresEngine, error) {
+func setEngine(t *testing.T, envVariables EnvVariables) alloydbutil.PostgresEngine {
 	t.Helper()
-	username, password, database, projectID, region, instance, cluster, _ := getEnvVariables(t)
 	ctx := context.Background()
 	pgEngine, err := alloydbutil.NewPostgresEngine(ctx,
-		alloydbutil.WithUser(username),
-		alloydbutil.WithPassword(password),
-		alloydbutil.WithDatabase(database),
-		alloydbutil.WithAlloyDBInstance(projectID, region, cluster, instance),
+		alloydbutil.WithUser(envVariables.Username),
+		alloydbutil.WithPassword(envVariables.Password),
+		alloydbutil.WithDatabase(envVariables.Database),
+		alloydbutil.WithAlloyDBInstance(envVariables.ProjectID, envVariables.Region, envVariables.Cluster, envVariables.Instance),
 	)
 	if err != nil {
 		t.Fatal("Could not set Engine: ", err)
 	}
 
-	return pgEngine, nil
+	return pgEngine
 }
 
-func setVectorStore(t *testing.T) (alloydb.VectorStore, func() error, error) {
+func vectorStore(t *testing.T, envVariables EnvVariables) (alloydb.VectorStore, func() error, error) {
 	t.Helper()
-	_, _, _, _, _, _, _, table := getEnvVariables(t)
-	pgEngine, err := setEngine(t)
-	if err != nil {
-		t.Fatal(err)
-	}
+	pgEngine := setEngine(t, envVariables)
 	ctx := context.Background()
 	vectorstoreTableoptions := alloydbutil.VectorstoreTableOptions{
-		TableName:     table,
+		TableName:     envVariables.Table,
```

Comment:
Add overwrite existing equal to true, since we call this multiple times and test error with "vectorstore_test.go:158: failed to create table: ERROR: relation "test-1233" already exists (SQLSTATE 42P07)"